### PR TITLE
Add lc-test.js

### PR DESCRIPTION
**Pull request by kazk**
Reduce boilerplate. More helpers can be added in the future.

### CODE REVIEW

File: example/test.js [hunk 1]
```diff
@@ -1,13 +1,9 @@
...
-import { solution } from "./files.js"; // /workspace/files.js
-
-LC.config.purity = "Let";
-LC.config.numEncoding = "Church";
-const toInt = LC.toIntWith(LC.config);
-const { counter } = LC.compile(solution());
+config.purity = "Let";
+config.numEncoding = "Church";
+const toInt = toIntWith(config);
+const { counter } = compile(solution());
```
Comment (kazk) [inline example/test.js]: Maybe `getSolution()` is better? Or `solutionCode`?

Comment (Kacarott) [reply example/test.js]: I think `getSolution()` or `readSolution()` is better. It makes it more natural if an author needs to work with a solution text more than just compiling, using `const solution = readSolution()`

File: workspace/test.js [hunk 2]
```diff
@@ -1,13 +1,9 @@
-import { assert, config as chaiConfig } from "chai";
-chaiConfig.truncateThreshold = 0;
+import { assert, config, toIntWith, compile, solution } from "./lc-test.js";
```
Comment (Kacarott) [inline workspace/test.js]: Does `./lc-test.js` also have to be local? (I suppose so, for the same reason as `./files.js`)

Comment (kazk) [reply workspace/test.js]: If you want, we can move `import.meta.url` to `test.js` instead:

```javascript
export const getSolution = (base) => readFileSync(new URL("./solution.lc", base), {encoding: "utf8"});
// or general
export const getRelative = (base, file) => readFileSync(new URL(file, base), {encoding: "utf8"});
```

```javascript
import { getSolution, getRelative } from "@codewars/lambda-calculus/test";

const { counter } = compile(getSolution(import.meta.url));

const { counter } = compile(getRelative(import.meta.url, "./solution.lc"));
```

Comment (Kacarott) [reply workspace/test.js]: Hmmm no that doesn't seem very pleasant. I think its fine with just `./lc-test.js`

File: example/test.js (hunk 1 above)
Comment (JohanWiltink) [reply example/test.js]: I hadn't realised `solution` would clash with `const solution = compile(solution())`, but I do that in several bigger kata.

`getSolution` has my vote.

I have a [proposal](https://discord.com/channels/846624424199061524/933847946808684596/943860712483221554) for `configure` in which `toInt` is differently than it is used here.

File: workspace/test.js (hunk 2 above)
Comment (JohanWiltink) [reply workspace/test.js]: I like `./lc-test.js`.

Would not `./lc-tests.js` completely obviate the need for a separate `./files.js`?

**Review by JohanWiltink — CHANGES_REQUESTED** [review]
Karott and I decided on some redesign; I'm implementing that now and I'll incorporate the workspace redesign as well. It's all interconnected or tests will fail.

File: workspace/test.js (hunk 2 above)
Comment (kazk) [reply workspace/test.js]: Yeah, `files.js` doesn't exist anymore.

**Comment by JohanWiltink** [comment]
I have tried to do everything from this PR in mine.

If I did that right, this one can be summarily rejected. I just don't know if I did.

**Review by JohanWiltink — APPROVED** [review]

**Pull request merged**